### PR TITLE
add in offline baseline code

### PR DESCRIPTION
**Pull request by cz0204**
add in all current work for offline baseline solution into comp-1. The code is added into path /SMARTS/competition/track-2 with two folders 'train' and 'submission'

### CODE REVIEW

File: competition/track-2/train/train.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        rewards = list()
+        terminals = list()
+        print('processing scenario ' + scenario)
+        vehicle_ids = list()
+        
+        for filename in client.listdir(path + scenario):
+            if filename.endswith('.png'):
+                vehicle_id = re.search('vehicle-(.*).png', filename).group(1)
+                if vehicle_id not in vehicle_ids:
+                    vehicle_ids.append(vehicle_id)
```
Comment (Gamenot) [inline competition/track-2/train/train.py:69]:
```suggestion
                match = re.search('vehicle-(.*).png', filename)
                if match is None:
                    continue
                vehicle_id = match.group(1)
                if vehicle_id not in vehicle_ids:
                    vehicle_ids.append(vehicle_id)
```

File: competition/track-2/train/train.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+import torch
+torch.cuda.empty_cache()
+import pathlib
+from d3rlpy.metrics.scorer import average_value_estimation_scorer
+from d3rlpy.metrics.scorer import td_error_scorer
+import glob
+
+
+
+path = '/net/storage-1/home/c84201475/waymo_bev/'
```
Comment (Gamenot) [inline competition/track-2/train/train.py:23]: Please remove this

File: competition/track-2/train/train.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        actions = list()
+        rewards = list()
+        terminals = list()
+        print('processing scenario ' + scenario)
+        vehicle_ids = list()
+        
+        for filename in client.listdir(path + scenario):
+            if filename.endswith('.png'):
+                match = re.search('vehicle-(.*).png', filename)
+                assert match is not None:
```
Comment (Gamenot) [inline competition/track-2/train/train.py:60]:
```suggestion
                assert match is not None
```

**Comment by Adaickalavan** [comment]
@cz0204 
+ I have attempted to remove unused code. Please verify the changes.
+ Consider writing some instructions on Track2 for participants.
+ Consider writing a `README.md` explaining how to use the training code.  
+ Consider writing a `README.md` explaining how to make a submission for Track2. If there are any specific interfaces or requirements which the participant submitted code must follow in order for you to re-train and re-test their code, consider specifying those interfaces and requirements in Codalab Track2 description.

**Pull request merged**